hack/deploy.sh

```diff
@@ -71,7 +71,7 @@ deploy_docker() {
   docker login -u $API_USERNAME -p $QUAY_PASS https://quay.io
   docker push quay.io/goswagger/swagger:$CIRCLE_TAG
 
-  if [[ -n "${EXTRA_TAG}" ]] ; then
+  if [[ -n "${EXTRA_TAG-""}" ]] ; then
```

Comment:
Why this script is not removed altogether? It seems no to be used any longer with this new CI.